models/domain.go

```diff
@@ -78,7 +78,6 @@ func (d *Domain) IsQueueable(db scanStore, list policyList) (bool, string, Scan)
 func (d *Domain) PopulateFromScan(scan Scan) {
 	// We should only trust MTA-STS info from a successful MTA-STS check.
 	if scan.Data.MTASTSResult != nil && scan.SupportsMTASTS() {
-		d.MTASTSMode = scan.Data.MTASTSResult.Mode
```

Comment:
We're using MTASTSMode in a slightly different way than in the scan. Maybe we could rename "MTASTSMode" to something else...

Reply:
It looks to me like we're just using it as a boolean field - I'd be into making it a boolean called `MTASTS` to match the database column. "on" is the value that the frontend will POST for the form element "mta-sts" if it the box is checked, we can set the boolean accordingly based on that in `getDomainParams`.

(I wonder if I introduced this because of some confusion between caching the mta-sts mode as a column on the scan table vs the domain. Looks like the only purpose is serving pre-this-PR is to capture the form value from `/add-domain` and do some logic in that handler based on the value.)